### PR TITLE
[DC-1078] Truncate FitBit Data

### PR DESCRIPTION
**Pull request by dcampbell-vumc**
* Created bash script `truncate_fitbit.sh` to run the `truncate_fitbit_data` cleaning rule
* Modified `TRUNCATE_FITBIT_DATA_QUERY` to account for participants who have submitted data before and after the cutoff date.

### CODE REVIEW

File: data_steward/tools/truncate_fitbit.sh [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+# create empty fitbit sandbox dataset
+sandbox_dataset="${fitbit_dataset}_sandbox"
+bq mk --dataset --description "${dataset_release_tag} sandbox dataset for ${fitbit_dataset}" "${APP_ID}":"${sandbox_dataset}"
+
+# Create logs dir
+LOGS_DIR="${DATA_STEWARD_DIR}/logs"
+mkdir -p "${LOGS_DIR}"
+
+# Apply cleaning rules
+python "${CLEANER_DIR}/truncate_fitbit_data.py" --project_id "${APP_ID}" --dataset_id "${fitbit_staging_dataset}" --sandbox_dataset_id "${sandbox_dataset}"kj.kj -s 2>&1 | tee -a "${LOGS_DIR}"/truncate_fitbit_log.txt
```
Comment (nishanthpp93) [inline data_steward/tools/truncate_fitbit.sh:76]: I'm not sure how the `kj.kj` showed up, but the rest looks fine

**Review by nishanthpp93 — COMMENTED** [review]
Everything looks good except for the one typo(?)

File: data_steward/tools/truncate_fitbit.sh (hunk 1 above)
Comment (dcampbell-vumc) [reply data_steward/tools/truncate_fitbit.sh]: I removed that weird typo!

**Review by nishanthpp93 — APPROVED** [review]
Looks good!

**Pull request merged**